Update Button docs (#843)
jpmorganchase/uitk: packages/core/stories/button.doc.stories.mdx

Icons can be put before or after text, or used alone.

#### Design considerations

> - An icon should be put before the text if it is further describing the button's action, such as upload, submit, play, done. This can be useful in busy environments where an additional description can improve speed to locate actions.

Reviewer: ```suggestion
- An icon should be put before the text if it is further describing the button's action, such as upload, submit, play, or done. This can be useful in busy environments where an additional description can improve speed in locating actions.
```

Author: I think 'done' is an example of a button label. I will update these to title case, with quote marks, to reflect that.

- When you put an icon after the button label, it typically suggests direction or movement, such as opening a menu overlay, expanding a panel or opening a document. In this case, when the button is pressed there is typically an immediate visual change.

> - A button with only an icon and no text should be self-explanatory, ideally using well known metaphors which are easily related to. If the purpose of the button isn’t clear, a label should be added. If space is a consideration, add a tooltip to the button set to display on hover after a short delay (typically 300ms, `--uitk-delay-perceptible`). The tooltip should add context to the action such as 'Edit this form', 'Print this document', 'Lock document state'. 

Reviewer: ```suggestion
- A button with only an icon and no text should be self-explanatory, ideally using well-known metaphors which are easily related to. If the purpose of the button isn’t clear, a label should be added. If space is a consideration, add a tooltip to the button set to display on hover after a short delay (typically 300ms, `--uitk-delay-perceptible`). The tooltip should add context to the action such as 'Edit this form', 'Print this document', and 'Lock document state'. 
```

Author: 'and' is not appropriate in the last sentence as it is a list of possible tooltips, I will add 'or'.

> - A button with only an icon and no text should be self-explanatory, ideally using well known metaphors which are easily related too. If the purpose of the button isn’t clear, a label should be added. If space is a consideration, add a tooltip to the button set to display on hover after a short delay (typically 300ms, `--foundation-delay-perceptible`). The tooltip should add context to the action such as 'Edit this form', 'Print this document', 'Lock document state'. 

Reviewer 2: ```suggestion
- A button with only an icon and no text should be self-explanatory, ideally using well known metaphors which are easily related too. If the purpose of the button isn’t clear, a label should be added. If space is a consideration, add a tooltip to the button set to display on hover after a short delay (typically 300ms, `--uitk-delay-perceptible`). The tooltip should add context to the action such as 'Edit this form', 'Print this document', 'Lock document state'. 
```
